assets/components/contributionSelection/contributionSelectionActions.js

```diff
@@ -0,0 +1,32 @@
+// @flow
+
+// ----- Imports ----- //
+
+import type { Contrib as ContributionType } from 'helpers/contributions';
+
+
+// ----- Types ----- //
+
+export type Action =
+  | { type: 'SET_CONTRIBUTION_TYPE', scope: string, contributionType: ContributionType }
+  | { type: 'SET_AMOUNT', scope: string, amount: string }
+  | { type: 'SET_CUSTOM_AMOUNT', scope: string, amount: string };
+
+
+// ----- Action Creators ----- //
+
+export default function contributionSelectionActionsFor(scope: string): Object {
```

Comment:
I would try to avoid `export default function` in favor of 
```
export {
   nameOfFunction
}
```
In that way when you import the function somewhere else you need to match the name with the real name of the functions and the code is more readable.